resources/views/Widgets/Item/OrderPropertyWidget.twig

```diff
@@ -12,7 +12,7 @@
 
 <div class="widget widget-order-property widget-{{ appearance }} {% if spacingMargin | length > 0 %} {{ spacingMargin }}{% endif %}{% if customClass | length > 0 %} {{ customClass }}{% endif %}" {% if inlineMargin | length > 0 %} style="{{ inlineMargin }}"{% endif %}>
     <order-property-list
-        {% if not isPreview %}v-if="$store.getters.currentItemVariation.filter.isSalable && $store.getters.variationGroupedProperties.length"{% endif %}
+        :is-preview="{{ isPreview | json_encode }}"
```

Comment:
Wozu wird isPreview benötigt? kann man in der Komponente nicht auf App.isShopBuilder prüfen?

Reply:
Kann raus